_topic_map.yml

```diff
@@ -279,6 +279,38 @@ Topics:
         Distros: openshift-enterprise
       - Name: Installing a Stand-alone Registry
         File: stand_alone_registry
+  - Name: Configuring Authentication
+    Dir: authentication
+    Distros: openshift-origin,openshift-enterprise
+    Topics:
+      - Name: Overview
+        File: index
+      - Name: Configuring Allow All Authentication
```

Comment:
I think all of these "Configuring"s could be dropped at the beginning of the various identity provider topics, just to clean up how it looks in the left-nav. But leave "Configuring" in the title of the `.adoc`s themselves.